Backend/app/rider-platform/rider-app/Main/src-read-only/Domain/Types/MerchantMessage.hs

```diff
@@ -48,6 +48,7 @@ data MessageKey
   | PRE_PICKUP_DELIVERY_RECEIVER
   | SEND_SCHEDULED_RIDE_DETAILS
   | SCHEDULED_RIDE_OTP
+  | METRO_TICKET_BOOKING_CANCELLED
```

Comment:
As this file is read-only you will have to add it via yaml files & `run-generator`, it will add it.

Reply:
Done

Reply:
I still don't see yaml file in the diff, did you push recent changes ?